key-visualizer-monitoring-tool.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Key Visualizer
+suammary: Learn the Key Visualizer feature.
+category: how-to
+---
+
+# Key Visualizer
+
+Key Visualizer is a feature for analyzing the usage of TiDB and troubleshooting the traffic hotspot. The feature generates a visual report of the metrics of the TiDB cluster over a period of time. With this visual report, you can quickly and intuitively observe the overall cluster hotspot and traffic distribution.
+
+## Access Key Visualizer
+
+Key Visualizer is one of the TiDB Dashboard components. It is directly integrated to PD instances and does not require a separate deployment. You can access this TiDB Dashboard component on any PD instance via browsers by using the following address:
+
+{{< copyable "" >}}
+
+```
+http://PDAddress:PDPort/dashboard
+```
+
+> **Note:**
+>
+> + If multiple PD instances exist, you can use any PD instance address to access TiDB Dashboard.
+> + By default, `PDPort` is `2379`. If you have modified the corresponding PD parameter, use the corresponding modified PD port.
+> + You need the `root` account of TiDB to log into TiDB Dashboard.
+
+## Interface demonstration
+
+The following image is a demonstration of the Key Visualizer interface:
+
+![Key Visualizer Interface](/media/dashboard/keyvisualizer/overview.png)
+
+From the interface above, you can see the following objects:
+
++ A large heatmap that shows changes of the overall traffic over time.
++ The detailed information of heatmap axises.
```

Comment:
It should be the information of a certain coordinate point, not the coordinate axis.